### PR TITLE
lspath: add page

### PR DESCRIPTION
**Pull request by clementi**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/lspath.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# lspath
+
+> Print the contents of the system PATH variable, with one element per line.
```
Comment (kbdharun) [inline pages/common/lspath.md:3]:
```suggestion
> CLI app to list the contents of the PATH environment variable, with optional paging.
```
How about this?

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have a suggestion for this page.

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/lspath.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+> CLI app to list the contents of the PATH environment variable, with optional paging.
+> More information: <https://github.com/clementi/lspath>.
+
+- Print the contents of the system PATH variable, with one element per line:
+
+`lspath`
+
+- Print the contents of the system PATH variable, with one element per line, with the output paged:
+
+`lspath -p`
```
Comment (kbdharun) [inline pages/common/lspath.md:12]:
```suggestion
`lspath --page`
```
Small suggestion, we generally prefer long arguments in tldr.

File: pages/common/lspath.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# lspath
+
+> CLI app to list the contents of the PATH environment variable, with optional paging.
+> More information: <https://github.com/clementi/lspath>.
+
+- Print the contents of the system PATH variable, with one element per line:
```
Comment (EmilySeville7cfg) [inline pages/common/lspath.md:6]:
```suggestion
- Print the current contents of the system PATH variable, with one element per line:
```

Optional fix: `the current` may sound better.

File: pages/common/lspath.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+`lspath`
+
+- Print the contents of the system PATH variable, with one element per line, with the output paged:
```
Comment (EmilySeville7cfg) [inline pages/common/lspath.md:10]:
```suggestion
- Print the current contents of the system PATH variable, with one element per line, with the output paged:
```

The same issue as above.

**Review by EmilySeville7cfg — APPROVED** [review]
- No mandatory fixes found, approved.
- This page should be reviewed by `lspath` users.
- Thanks for a contribution! 😄

**Pull request merged**